### PR TITLE
adds alias for find_by_id in datatore book model

### LINKED ISSUE #39: 6-task-queueing with datastore: NoMethodError: undefined method `find_by_id' for Book:Class
I receive the following error when the workers run (using `backend:datastore` for `6-task-queueing`. 

```
13:26:49 worker.2       | Book lookup request (5679413765079040)
13:26:49 worker.2       | rake aborted!
13:26:49 worker.2       | NoMethodError: undefined method `find_by_id' for Book:Class
13:26:49 worker.2       | 6-task-queueing/lib/active_job/queue_adapters/pub_sub_queue_adapter.rb:40:in `block in run_worker!'
13:26:49 worker.2       | 6-task-queueing/vendor/bundle/ruby/2.2.0/gems/gcloud-0.4.0/lib/gcloud/pubsub/subscription.rb:376:in `block (2 levels) in listen'
13:26:49 worker.2       | 6-task-queueing/vendor/bundle/ruby/2.2.0/gems/gcloud-0.4.0/lib/gcloud/pubsub/subscription.rb:376:in `each'
13:26:49 worker.2       | 6-task-queueing/vendor/bundle/ruby/2.2.0/gems/gcloud-0.4.0/lib/gcloud/pubsub/subscription.rb:376:in `block in listen'
13:26:49 worker.2       | 6-task-queueing/vendor/bundle/ruby/2.2.0/gems/gcloud-0.4.0/lib/gcloud/pubsub/subscription.rb:373:in `loop'
13:26:49 worker.2       | 6-task-queueing/vendor/bundle/ruby/2.2.0/gems/gcloud-0.4.0/lib/gcloud/pubsub/subscription.rb:373:in `listen'
13:26:49 worker.2       | 6-task-queueing/lib/active_job/queue_adapters/pub_sub_queue_adapter.rb:36:in `run_worker!'
13:26:49 worker.2       | 6-task-queueing/lib/tasks/run_worker.rake:4:in `block in <top (required)>'
```

This is fixed by editing `lib/active_job/queue_adapters/pub_sub_queue_adapter.rb` and changing `find_by_id` to `find`

### PR DESCRIPTION
**Pull request by bshaffer**
fixes #39

### CODE REVIEW

**Comment by bshaffer** [comment]
@remi @jeffmendoza PTAL

File: 6-task-queueing/structured_data/datastore/app/models/book.rb [hunk 1]
```diff
@@ -74,6 +74,11 @@ def self.find id
     from_entity entities.first if entities.any?
   end
 
+  # an alias for "find" for compatibility with Active Record
+  def self.find_by_id id
+    self.find id
```
Comment (remi) [inline 6-task-queueing/structured_data/datastore/app/models/book.rb]: just `find id`, no `self.`

Comment (remi) [inline 6-task-queueing/structured_data/datastore/app/models/book.rb]: actually, use `alias_method`

Comment (remi) [inline 6-task-queueing/structured_data/datastore/app/models/book.rb]: (then LGTM!)

**Comment by bshaffer** [comment]
@remi I used `singleton_class.send` because it's a class method. PTAL

File: 6-task-queueing/structured_data/datastore/app/models/book.rb (hunk 1 above)
Comment (bshaffer) [inline 6-task-queueing/structured_data/datastore/app/models/book.rb]: updated, PTAL

**Pull request merged**